src/js/index.js

```diff
@@ -53,9 +55,18 @@ RequestUtil.json = function(options = {}) {
 (function() {
   function renderApplication() {
     function renderAppToDOM(content) {
-      ReactDOM.render(content, domElement, function() {
-        PluginSDK.Hooks.doAction("applicationRendered");
-      });
+      ReactDOM.render(
+        <React.Fragment>
+          <MountService.Mount type="General:PageTitle">
+            <Helmet titleTemplate={"%s - " + i18nMark("Mesosphere DC/OS")} />
```

Comment:
Shouldn't it be "D2IQ" now?

Reply:
I think Mesosphere is intentional here. DC/OS is a "Mesosphere" product. Kommander and Konvoy are "Ksphere" products.

Somebody please correct me if I'm wrong